### ISSUE TITLE
[Bug] Quoting error with PostgreSQL 9.6 as database

### ISSUE THREAD

**Issue by peteeckel** [grafana/grafana#7639]
Hi, 

In my PostgreSQL log I find recurring error messages like this: 

```
< 2017-02-22 10:48:01.408 UTC > LOG:  connection received: host=192.168.106.101 port=55590
< 2017-02-22 10:48:01.490 UTC > LOG:  connection authorized: user=grafana database=grafana SSL enabled (protocol=TLSv1.2, cipher=ECDHE-RSA-AES256-GCM-SHA384, compression=off)
< 2017-02-22 10:48:01.529 UTC > ERROR:  relation "INFORMATION_SCHEMA.TABLES" does not exist at character 26
< 2017-02-22 10:48:01.529 UTC > STATEMENT:  SELECT "TABLE_NAME" from "INFORMATION_SCHEMA"."TABLES" WHERE "TABLE_SCHEMA"=$1 and "TABLE_NAME"=$2
< 2017-02-22 10:48:01.535 UTC > ERROR:  relation "INFORMATION_SCHEMA.TABLES" does not exist at character 26
< 2017-02-22 10:48:01.535 UTC > STATEMENT:  SELECT "TABLE_NAME" from "INFORMATION_SCHEMA"."TABLES" WHERE "TABLE_SCHEMA"=$1 and "TABLE_NAME"=$2
```

This can be reproduced by restarting grafana-server, which immediately triggers the error messages in the PostgreSQL log. 

The root cause seems to be the quoting of the schema/table names in Grafana's query, as can be reproduced in psql:

```
grafana=> SELECT "TABLE_NAME" from "INFORMATION_SCHEMA"."TABLES";
ERROR:  relation "INFORMATION_SCHEMA.TABLES" does not exist
LINE 1: SELECT "TABLE_NAME" from "INFORMATION_SCHEMA"."TABLES";
```
while
```
grafana=> SELECT TABLE_NAME from INFORMATION_SCHEMA.TABLES limit 1;
  table_name   
---------------
 migration_log
(1 row)
```
gives the correct result. So the issue obviously is with the quoting of the names (all of them, table, row and schema name must not be quoted according to my tests). 

Grafana Version 4.1.2
Graphite 0.9.15
PostgreSQL 9.6.2
CentOS 7.3

Cheers, 

  Pete.

**Comment by peteeckel** [comment]
This is still present in 4.3 (just for the record)

**Comment by torkelo** [comment]
It was never fixed, it still open

**Comment by max-neverov** [comment]
The issue was fixed by #9460.
Not reproducible on Grafana v6.0.0-pre3 (989147a), postgres 11.1-alpine, macos.

**Comment by marefr** [comment]
Thanks @max-neverov
